hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py

```diff
@@ -915,6 +1049,18 @@ def getReferenceRPY(self, lname, frame_name='WAIST'):
             return HrpsysConfigurator.getReferenceRPY(self, lname, frame_name)
     
     def setTargetPose(self, gname, pos, rpy, tm, frame_name='WAIST'):
+        '''!@brief
+        Move the end-effector to the given absolute pose.
+        All d* arguments are in meter.
+
+        @param gname str: Name of the joint group.
+        @param pos list of float: In meter.
+        @param rpy list of float: In radian.
+        @param tm float: Second to complete the command.
+        @param frame_name str: Name of the frame that this particular command
+                           references to.
+        @return bool: False if unreachable.
+        '''
```

Comment:
need to add version check at -> https://github.com/fkanehiro/hrpsys-base/blob/master/python/hrpsys_config.py#L1339